cmd/apmtool/main.go

```diff
@@ -81,6 +81,14 @@ func main() {
 				Destination: &commands.cfg.APMServerURL,
 				Persistent:  true,
 			},
+			&cli.BoolFlag{
+				Name:        "insecure",
+				Usage:       "sets agents to skip the server's TLS certificate verification",
```

Comment:
```suggestion
				Usage:       "skip TLS certificate verification of Elasticsearch and APM server",
```

I'm thinking why isn't `ELASTIC_APM_VERIFY_SERVER_CERT=0` used, and realized that this setting is used for both ES and APM communication. In that case, can we state that explicitly in the usage that it is for both?

Reply:
sure, thanks for the suggestion!